apps/akasha/src/components/post-page/post-page.tsx

```diff
@@ -268,16 +268,16 @@ const PostPage: React.FC<IPostPageProps & RootComponentProps> = props => {
       <Helmet>
         <title>Post | Ethereum World</title>
       </Helmet>
-      {postReq.status === 'loading' && <EntryCardLoading />}
-      {postReq.status === 'error' && (
```

Comment:
I don'w understand these changes, are they part of a fix? postReq.status === 'loading' is the same as postReq.isLoading. isLoading is actually derived from the status state.
My point is that this is just a matter of preference and it does not affect the functionality in any way.

Reply:
True, not part of the fix, but I was under the impression we were slowly refactoring away from string literal checks to boolean flag checks when we see them.